ui/app/components/keyboard-shortcuts-modal.js

```diff
@@ -49,4 +55,8 @@ export default class KeyboardShortcutsModalComponent extends Component {
   untetherFromElement(self, _, { hint }) {
     hint.binder.destroy();
   }
+
+  @action toggleListener() {
+    this.keyboard.enabled = !this.keyboard.enabled;
+  }
```

Comment:
`question`: Are we using the `localStorageProperty` API correctly here? I notice we're using this action instead of the `set` method on the API. Does this save to `localStorage` correctly?

Reply:
Yep! Setting a property in this way is an implicit `set` for computed macros. I think they probably use JS proxies under the hood.